op-batcher/readme.md

```diff
@@ -137,6 +137,185 @@ transaction. But in the case of a DA backlog (as defined by OP_BATCHER_THROTTLE_
 block builder to instead impose a (tighter) block level limit of OP_BATCHER_THROTTLE_BLOCK_SIZE, and a single
 transaction limit of OP_BATCHER_THROTTLE_TRANSACTION_SIZE.
 
+### Enhanced DA Throttling Mechanisms
+
+The batcher now supports multiple advanced throttling strategies beyond simple binary on/off throttling. These controllers dynamically adjust throttling intensity based on the current DA backlog, providing more nuanced control over data availability management.
+
+#### Throttling Controller Types
+
+The batcher supports four throttling controller types, each with different response characteristics:
+
+**Step Controller** (Default)
+- **Behavior**: Binary on/off throttling
+- **Response**: No throttling below threshold, maximum throttling above threshold
+- **Use Case**: Simple, predictable throttling behavior
+
+**Quadratic Controller**
+- **Behavior**: Quadratically scales throttling intensity (slower initial response, faster at high loads)
+- **Response**: Gentle at low overload, aggressive at high overload
+- **Use Case**: Tolerates brief spikes while responding strongly to sustained overload
+
+**PID Controller**
+- **Behavior**: Proportional-Integral-Derivative control with configurable parameters
+- **Response**: Sophisticated control with predictive and corrective components
+- **Use Case**: Complex load patterns requiring precise control and minimal overshoot
+
+#### Visual Representation of Throttling Responses
+
+```
+Throttling Intensity (0.0 = No Throttling, 1.0 = Maximum Throttling)
+
+Step Controller:
+    |
+1.0 |     ████████████████████████████████
+    |     │
+0.5 |     │
+    |     │
+0.0 |█████│
+    +─────┼────────────────────────────────
+         threshold              load →
+
+Quadratic Controller:
+    |
+1.0 |                     ████
+    |                 ████
+0.5 |             ████
+    |         ████
+0.0 |████████████
+    +─────┼─────────────┼────────────────────
+         threshold    max_threshold  load →
```

Comment:
I struggle to recognise this as a quadratic curve. I think we should just commit a png with a line drawing instead of trying to draw it like this.